star/defaults/controls.defaults

```diff
@@ -11216,7 +11216,7 @@
       ! stop_for_bad_nums
       ! ~~~~~~~~~~~~~~~~~
 
-      ! If true and report_ierr is also true, then stop for bad numbers (NaNs or infinity).
+      ! If true and ``report_ierr`` is also true, then stop for bad numbers (NaNs or infinity).
```

Comment:
At a glance this doesn't look like what we've changed the behaviour to. Instead it looks like the correct description of the new behaviour (which I prefer and support) is
```suggestion
      ! If true, then stop for bad numbers (NaNs or infinity).
```
I.e., `stop_for_bad_nums` now stops whether or not `report_ierr` is true because the `stop_for_bad_nums` checks our outside the `if (report_ierr)` blocks.

Reply:
Oh! Very good point! I'll make the change.  `stop_for_bad_nums` can simply function independently from having diagnostics switched on (as it already does in most cases)